bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -105,49 +120,54 @@ pub mod pallet {
 		/// Convert versioned location failure
 		UnsupportedLocationVersion,
 		/// Check location failure, should start from the dispatch origin as owner
-		OwnerCheck,
+		InvalidAssetOwner,
 		/// Send xcm message failure
 		Send,
 		/// Withdraw fee asset failure
 		FundsUnavailable,
+		/// Convert to reanchored location failure
+		LocationConversionFailed,
+		/// The desired destination was unreachable, generally because there is a no way of routing
+		/// to it.
+		Unreachable,
+	}
+
+	impl<T: Config> From<SendError> for Error<T> {
+		fn from(e: SendError) -> Self {
+			match e {
+				SendError::Fees => Error::<T>::FundsUnavailable,
+				SendError::NotApplicable => Error::<T>::Unreachable,
+				_ => Error::<T>::Send,
```

Comment:
Mirror inbound-queue-v2 error types:

```suggestion
				_ => Error::<T>::SendFailure,
```

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/6/commits/04eac741164ea28a7d76badd171fe6e4335e3dec